### PR TITLE
Feature/client edit my profile

### PR DESCRIPTION
**Pull request by ZuzaTwardowska**
image

### CODE REVIEW

File: Frontend/apps/client-catering-app/src/pages/MainPage.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const MainPage = (props: MainPageProps) => {
 
   useEffect(() => {
     service.execute!(
-      getDietsConfig(userContext?.authApiKey!),
+      getDietsConfig(userContext?.authApiKey!, query),
```
Comment (jtarkowski27) [inline Frontend/apps/client-catering-app/src/pages/MainPage.tsx:35]: Please revert change in this file

File: Frontend/apps/client-catering-app/src/Services/URLcreator.tsx [hunk 2]
```diff
@@ -4,10 +4,12 @@ export const getRegisterClientURL = (): string => apiUrl + "/Client/register";
 
 export const getLoginClientURL = (): string => apiUrl + "/Client/login";
 
-export const getDietsURL = (): string => apiUrl + "/Diets";
+export const getDietsURL = (query: string): string => apiUrl + "/Diets" + query;
```
Comment (jtarkowski27) [inline Frontend/apps/client-catering-app/src/Services/URLcreator.tsx:7]: Revert this

File: Frontend/apps/client-catering-app/src/Services/configCreator.tsx [hunk 3]
```diff
@@ -13,11 +14,11 @@ export const getRegisterConfig = (): ApiConfig =>
 export const getLoginConfig = (): ApiConfig =>
   ({ method: "post", header: "", url: getLoginClientURL() } as ApiConfig);
 
-export const getDietsConfig = (key: string) =>
+export const getDietsConfig = (key: string, query: string) =>
```
Comment (jtarkowski27) [inline Frontend/apps/client-catering-app/src/Services/configCreator.tsx:17]: revert this as well

File: Frontend/apps/client-catering-app/src/Services/configCreator.tsx [hunk 4]
```diff
@@ -13,11 +14,11 @@ export const getRegisterConfig = (): ApiConfig =>
...
   ({
     method: "get",
     header: { Authorization: "Bearer " + key },
-    url: getDietsURL(),
+    url: getDietsURL(query),
```
Comment (jtarkowski27) [inline Frontend/apps/client-catering-app/src/Services/configCreator.tsx]: same here

File: Frontend/apps/client-catering-app/src/pages/MainPage.tsx (hunk 1 above)
Comment (ZuzaTwardowska) [reply Frontend/apps/client-catering-app/src/pages/MainPage.tsx]: done

File: Frontend/apps/client-catering-app/src/Services/URLcreator.tsx (hunk 2 above)
Comment (ZuzaTwardowska) [reply Frontend/apps/client-catering-app/src/Services/URLcreator.tsx]: done

File: Frontend/apps/client-catering-app/src/Services/configCreator.tsx (hunk 3 above)
Comment (ZuzaTwardowska) [reply Frontend/apps/client-catering-app/src/Services/configCreator.tsx]: done

**Review by jtarkowski27 — APPROVED** [review]

**Pull request merged**